### DISCUSSION TITLE
Can we get an update to `pony-cause`?

### DISCUSSION THREAD

**Discussion by SpencerKaiser** [sequelize/umzug#561] [General] [answered]
Looks like there's a bug with the `Error` type within TypeScript ^4.6 that causes a typecheck error within `pony-cause`

[The maintainer of that repo has released a new version with a fix](https://github.com/voxpelli/pony-cause/issues/35#issuecomment-1201732554) for that specific issue (`2.1.0`). Any chance we could get an update to resolve that error?

**Comment by WikiRik** [comment, marked as answer]
`pony-cause` v2 drops support for Node 12 which is still supported by `umzug` v3 so updating `pony-cause` should be done in the next major release of `umzug`

**Reply by voxpelli** [reply]
Oh, didn't realize that would be a problem, I can backport the fix for you, would that help?

**Reply by WikiRik** [reply]
If that's not too much trouble for you that would be great. Thanks!

**Reply by SpencerKaiser** [reply]
Yall are awesome, thanks for the quick responses!!!

**Reply by voxpelli** [reply]
@WikiRik As it was mainly the build tool used to add ESM support besides the CJS files that required Node 14, I simply extended the support of 2.x to Node 12: https://github.com/voxpelli/pony-cause/releases/tag/v2.1.2

**Reply by mmkal** [reply]
Released [v3.1.2](https://github.com/sequelize/umzug/releases/tag/v3.1.2) with pony-cause v2.1.2!

**Reply by WikiRik** [reply]
Thanks @mmkal ! I can't mark replies as the answer but I'll mark this entire thread as answer

**Reply by SpencerKaiser** [reply]
@mmkal are you going to bring that change into `3.2.x` as well??

**Reply by mmkal** [reply]
@SpencerKaiser 3.2.x should also have pony cause ^2.1.2 as well, are you seeing something different?

**Reply by SpencerKaiser** [reply]
@mmkal I didn't look into it, I was just going off of what you said about `3.1.2`; I'll take your word for it! Thanks for the follow up, I'll check in with the maintainer of Mikro-orm to get that updated there as well!
